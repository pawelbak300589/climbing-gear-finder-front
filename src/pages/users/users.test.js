import React from 'react';
import { shallow } from "enzyme";

import UsersPage from "./users.component";

it('should render UsersPage component', () => {
    expect(shallow(<UsersPage />)).toMatchSnapshot();
});