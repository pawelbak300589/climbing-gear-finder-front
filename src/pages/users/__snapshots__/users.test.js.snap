// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render UsersPage component 1`] = `ShallowWrapper {}`;
